Source/WebKit/Shared/RemoteLayerTree/DynamicContentScalingImageBufferBackend.mm

```diff
@@ -84,7 +84,7 @@ static CFDictionaryRef makeContextOptions(const DynamicContentScalingImageBuffer
 
 DynamicContentScalingImageBufferBackend::DynamicContentScalingImageBufferBackend(const Parameters& parameters, const WebCore::ImageBufferCreationContext& creationContext, WebCore::RenderingMode renderingMode)
     : ImageBufferCGBackend { parameters }
-    , m_resourceCache(bridge_id_cast(adoptCF(RECGCommandsCacheCreate(nullptr))))
+    , m_resourceCache(creationContext.dynamicContentScalingResourceCache)
     , m_renderingMode(renderingMode)
 {
 }
```

Comment:
Shouldn't we assert here `ASSERT(m_resourceCache);`? Or is it okay to have `DynamicContentScalingImageBufferBackend` with a null `m_resourceCache`?

Reply:
It's fine to have one without a resource cache, it'll just send ports for images every time.